reverse-inbound.md

```diff
@@ -23,3 +28,7 @@ host:port. If no size is specified then only one such connection is created per
 
 After the connection has succeeded, this connection now works as it was a normal *inbound* connection.
 
+If `maxConnectionBackoffMillis` is specified for a reverse inbound connection then that value will be the maximum
+connection backoff time if the target host isn't available. When nothing is configured a default value of 30 seconds for
+maximum backoff time will be used. The backoff is exponential and will start att 1/10 of the maximum value and increase
```

Comment:
It is not exponential, it is linear, 30s would be 3s, 6s, 9s,12s... with a simple exponential backoff you use a fixed factor of 2 to multiple some amount for each retry up until a limit, say 3s, 6s, 12s, 24s... 
Which do we want to implement here?

Reply:
The implementation isn't linear, it goes from 1/10th of the max, then 1/9th, 1/8th ... until it reaches the max with 1/1th of the max which is the max value. Basically the simplest exponential-like algorithm I could cook up that could deterministically reach the maxBackoff in n steps where n was arbitrarily chosen as 10. The documentation just isn't clear and maybe should be more precise if the current backoff algorithm is used. The actual code probably has a better comment describing how the backoff works

Reply:
ah, i read the comment on line 110 `// Backoff in steps of 1/10, 2/10 .. 8/10, 9/10 until maxBackoffMillis is reached`
which is linear, and missed the actual implementation was different. :) If you update the comment and change the documentation here accordingly as either way it's not really exponential.

Reply:
I actually though that comment was correct, but it really isn't. Realized it while reworking the code now as well. I'll update the documentation and code-comment to be correct to not call it exponential since it really isn't and then describe what the backoff-algorithm really does